src/main/java/org/kairosdb/datastore/cassandra/CassandraDatastore.java

```diff
@@ -881,6 +886,11 @@ private void collectFromRowKeyIndex(List<DataPointsRowKey> collector,
                                         final long endTime,
                                         final SetMultimap<String, String> filterTags,
                                         final int limit) {
+        Span span = GlobalTracer.get().activeSpan();
+        if (span != null) {
+            span.setTag("index_name", "row_key");
```

Comment:
since you used the full table name above... I was expecting "row_key_index" here.

Reply:
aha, makes sense.